src/ucp/core/ucp_context.h

```diff
@@ -381,7 +381,12 @@ ucp_tl_iface_latency(ucp_context_h context, const uct_iface_attr_t *iface_attr)
 static UCS_F_ALWAYS_INLINE double
 ucp_tl_iface_bandwidth(ucp_context_h context, const uct_ppn_bandwidth_t *bandwidth)
 {
-    return bandwidth->dedicated + (bandwidth->shared / context->config.est_num_ppn);
+    /* BW is calculated as sum dedicated BW (doesn't depend from PPN) and
+     * (shared_BW) / PPN. In case if PPN == 2 then there is only one channel
+     * between EP and shared BW should not be divided by PPN */
```

Comment:
but if both processes send, they are sharing the channel
also it's not applicable for network transports

Reply:
cancel PR?

Reply:
yes, unless it fixes a real problem or performance issue?

Reply:
KNEM is never used due to shared_BW/2+